Submissions/pravinshedbale.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Pravin Shedbale",
+    "img": "[YOUR_IMG_URL]",
```

Comment:
Hello @pravinshedbale , please add your image url. Go through the steps listed in order for you to be able to add your image url.